docs/v1/index.bs

```diff
@@ -2050,6 +2050,36 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 </section>
 
 <section>
+  <h2 id="iana-considerations">IANA Considerations</h2>
+  
+  <p>The following registrations are for community review and will be submitted to the <a href="https://www.ietf.org/iesg/">IESG</a> for review, approval, and registration with <a href="https://www.iana.org/">IANA</a>.</p>
+  
+  <section>
+    <h3 id="iana-registration-service-worker">HTTP Header Field Service-Worker</h3>
+    <ul>
+        <li>Header Field Name: Service-Worker</li>
+        <li>Applicable Protocol: Hypertext Transfer Protocol (HTTP)</li>
+        <li>Status: Standard</li>
+        <li>Author/Change controller: W3C</li>
+        <li>Specification document(s): https://www.w3.org/TR/service-workers-1/</li>
```

Comment:
Not sure which document it should point to. Service Workers 1 is a snapshot of Service Workers which is a living document.